cnd/src/db/mod.rs

```diff
@@ -0,0 +1,25 @@
+pub mod models;
+pub mod schema;
+
+use self::models::NewSwap;
+use diesel::{prelude::*, sqlite::SqliteConnection};
+use dotenv::dotenv;
+use schema::swaps;
+use std::env;
+
+pub fn establish_connection() -> SqliteConnection {
+    dotenv().ok();
+
+    let database_url = env::var("DATABASE_URL").expect("DATABASE_URL must be set");
```

Comment:
May I suggest to default it to the config path `comit/cnd.sql3` and allow to override it via command line?

Reply:
This is done now using directories crate but I have not added a config option (either in file or command line).

Reply:
@D4nte Is the _config_ folder really the best place for this? `directories-rs` also has a `data_dir` method if I remember correctly. Could use that one?

Reply:
Anyway, that is bike-shedding. Let's try and get it working first :)

Reply:
I used cache dir.  Confusingly there is a data directory and a cache directory.  I belive cache is the correct choice.

Reply:
Hmm, according to [this](https://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html), the cache dir is for: 

> user specific non-essential data

In full: 

>  $XDG_CACHE_HOME defines the base directory relative to which user specific non-essential data files should be stored. If $XDG_CACHE_HOME is either not set or empty, a default equal to $HOME/.cache should be used. 

Now, I guess it is a question of whether we consider the swap history to be non-essential. If you are in the middle of a swap, loosing the database could be equivalent to loosing money. I think it would be surprising to the user to loose his swaps if they delete the cache directory. Also, on Windows, the `data_dir` property resolves to the `AppData\Roaming` folder which, for example, is [kept in sync](https://blogs.windows.com/windowsdeveloper/2016/05/03/getting-started-with-roaming-app-data) across all Windows installations of a user for UWP Apps (Universal Windows Platform Apps).

I'd suggest the `data_dir` folder since we are storing that is is not recoverable. Caches sound like you can rebuild them from something whereas loosing your swap database means they are gone.

Reply:
Good investigation on cache vs data @thomaseizinger.  Thanks for this, used data dir as suggested.